src/vs/workbench/contrib/comments/browser/commentsEditorContribution.ts

```diff
@@ -44,9 +48,10 @@ export class NextCommentThreadAction extends EditorAction {
 }
 
 export class PreviousCommentThreadAction extends EditorAction {
+	static ID: string = 'editor.action.previousCommentThread';
 	constructor() {
 		super({
-			id: 'editor.action.previousCommentThreadAction',
+			id: ID,
```

Comment:
@alexr00 it appears these aren't registered - the `lookupKeybinding` call returns `undefined`. same for the `next` one. I tried converting it to be an `Action2` and that didn't fix it either. I think maybe we'd need to do what you do here https://github.com/microsoft/vscode/blob/e49c56aab86fc41bae923b90962cc4cdc2de622a/src/vs/workbench/contrib/comments/browser/commentsEditorContribution.ts#L141-L153

as that one works?

Reply:
done